topics/metagenomics/tutorials/beer-data-analysis/tutorial.md

```diff
@@ -0,0 +1,391 @@
+usually require informatics background. [Galaxy](https://galaxyproject.org/) is
+an open-source platform for data analysis that enables users to use bioinformatics
+tools through its graphical web interface, accessible via any Web browser.
+
+So, in this tutorial, we will use Galaxy to extract and visualize the community
+of yeasts from a beer bottle.
+
+> ### Agenda
+>
+> In this tutorial, we will cover:
+>
+> 1. TOC
+> {:toc}
+>
+{: .agenda}
+
+
+# Prepare Galaxy and data
+
+First of all, this turorial will get you hands on the basic Galaxy tasks, including creating a history and importing data.
+
+## Get familier with Galaxy
+
+> ### {% icon hands_on %} Hands-on: Open Galaxy
+> 1. Open your favorite browser (Chrome, Safari or Firefox as your browser, not Internet Explorer!)
+> 2. Browse to a Galaxy instance: the one recommended by your instructor or one in the list **Galaxy instance** on the head of this page
+> 3. Log in or register (top panel)
+>
+>    ![Login or Register on the top panel](./images/login_register.png)
+{: .hands_on}
+
+The Galaxy homepage is divided into three panels:
+* Tools on the left
+* Viewing panel in the middle
+* History of analysis and files on the right
+
+![Galaxy interface screenshot showing history panel on the right, tools panel on the left, and main panel at the center](./images/galaxy_interface.png "The Galaxy interface")
+
+The first time you use Galaxy, there will be no files in your history panel.
+
+Any analysis should get its own Galaxy history. So let's start by creating a new one:
+
+> ### {% icon hands_on %} Hands-on: Prepare the Galaxy history
+>
+> 1. Create a new history for this analysis
+>
+>    {% snippet faqs/galaxy/histories_create_new.md %}
+>
+> 2. Rename the history
+>
+>    {% snippet faqs/galaxy/histories_rename.md %}
+>
+{: .hands_on}
+
+## Get data
+
+Before we can begin any Galaxy analysis, we need to upload the input data: FASTQ files
+
+> ### {% icon hands_on %} Hands-on: Upload your dataset
+>
+> 1. Import the sequenced data
+>
+>    - Option 1 [{% icon video %}](https://youtu.be/FFCDx1rMGAQ): Your own local data using **Upload Data** (recommended for 1-10 datasets).
+>
+>      {% snippet faqs/galaxy/datasets_upload.md %}
+>
+>    - Option 2: From Zenodo, an external server via URL
+>
+>      ```text
+>      {{ page.zenodo_link }}
+>      ```
+>
+>      {% snippet faqs/galaxy/datasets_import_via_link.md %}
+>
+>    Your uploaded file is now in your current history. When the file has uploaded to Galaxy, it will turn green. But, what is this file?
+>
+> 2. Click on the {% icon galaxy-eye %} (eye) icon next to the dataset name, to look at the file content
+>
+{: .hands_on}
+
+The contents of the file will be displayed in the central Galaxy panel.
+
+This file contains the sequences, also called **reads**, of DNA, *i.e.* sucession of nucleotides, for all fragments from the yeasts in the beer, in FASTQ format.
+
+> ### {% icon details %} FASTQ format
```

Comment:
I like this explanation, and I think it would be useful in other tutorials as well, would you mind making it an FAQ? maybe in the sequence-analysis topic? then we can include it in different tutorials :)